201-vmss-ubuntu-web-ssl/README.md

```diff
@@ -0,0 +1,149 @@
+# Create an SSL enabled Web server farm with VM Scale Sets

```

Comment:
Could you add the deploy to azure and visualize buttons? example here: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/sample-README.md